.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -27,7 +27,7 @@ implementation proposal in the referenced issues.
 
 ...
 
-**Release Notes: _(optional)_**
```

Comment:
Release notes are still optional. Not all pull requests change something that is worth mentioning in the release notes.

Reply:
- Not all of the sections are mandatory
- The `optional` does not belong in the heading